src/app/zap-templates/zcl/data-model/chip/account-login-cluster.xml

```diff
@@ -25,22 +25,22 @@ limitations under the License.
     <server init="false" tick="false">true</server>
     <description>This cluster provides commands that facilitate user account login on a Content App or a node. For example, a Content App running on a Video Player device, which is represented as an endpoint (see [TV Architecture]), can use this cluster to help make the user account on the Content App match the user account on the Client.</description>
 
-    <command source="client" code="0x00" name="GetSetupPIN" optional="false">
+    <command source="client" code="0x00" name="GetSetupPINRequest" response="GetSetupPINResponse" optional="false">
```

Comment:
I know this is just a renaming PR, but can you please file a followup to mark the relevant commands here as needing a timed interaction, per spec?

Reply:
All the commands that have "T" in the access column in the spec need to have `mustUseTimedInvoke="true"` in the XML.

Reply:
Aha ok. Added.

Reply:
To the GetSetupPin, Login and Logut.